src/test/java/api/loans/agetolost/ScheduledAgeToLostFeeChargingApiTest.java

```diff
@@ -289,6 +295,31 @@ public void shouldCloseLoanWhenNoFeesToCharge() {
     assertThatPublishedLoanLogRecordEventsAreValid();
   }
 
+  @Test
+  public void shouldNotAgeToLostDeclaredLostItem() {
+    final double declaredLostProcessingFee = 10.00;
+    useLostItemPolicy(lostItemFeePoliciesFixture.create(
+      lostItemFeePoliciesFixture.ageToLostAfterOneMinutePolicy()
+        .withPatronBilledAfterAgedLost(Period.weeks(1))
+        .withNoChargeAmountItem()
+        .doNotChargeProcessingFeeWhenAgedToLost()
+        .chargeProcessingFeeWhenDeclaredLost(declaredLostProcessingFee)).getId());
+
+    final ItemResource item = itemsFixture.basedUponNod();
+    final CheckOutResource checkOut = checkOutFixture.checkOutByBarcode(item);
+
+    declareLostFixtures.declareItemLost(forLoan(checkOut.getId()));
+
+    ageToLostFixture.ageToLostAndChargeFees();
+
+    final IndividualResource loanFromStorage = loansStorageClient.get(checkOut.getId());
+    assertThat(loanFromStorage.getJson(), hasNoDelayedBillingInfo());
+    assertThat(loanFromStorage.getJson(), LoanMatchers.isOpen());
+    assertThat(loanFromStorage, hasLostItemProcessingFee(isOpen(declaredLostProcessingFee)));
```

Comment:
Is there a need to check for the absence of other fees / fines?

Reply:
Make sense, I've added a check to verify that no lost item fee is assigned.